src/devices/Ds3231/Ds3231.cs

```diff
@@ -0,0 +1,176 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Device.I2c;
+using System.Device.I2c.Drivers;
+using System.Runtime.InteropServices;
+
+namespace Iot.Device.Ds3231
+{
+    /// <summary>
+    /// Realtime Clock DS3231
+    /// </summary>
+    public class Ds3231 : IDisposable
+    {
+        #region Address
+        private const byte RTC_I2C_ADDR = 0x68;
```

Comment:
After doing what @krwq suggested (changing the constructor so that it takes an I2cDevice instead) then I believe that this default address should be public, so that consumers won't have to memorize the address.